spec/models/hmpps_api/offender_spec.rb

```diff
@@ -219,24 +219,28 @@
   end
 
   describe '#over_18?' do
+    subject {
+      build(:offender, dateOfBirth: dob.to_s)
+    }
+
     context 'with a date of birth 50 years ago' do
```

Comment:
This context block is wrong – it seems worth fixing whilst we're here:

**~50 years ago~ 18 years ago**